codegen/src/test/resources/software/amazon/awssdk/codegen/poet/model/alltypesrequest.java

```diff
@@ -934,12 +919,7 @@ public final void setListOfMaps(Collection<Map<String, String>> listOfMaps) {
 
         @SafeVarargs
         public final void setListOfMaps(Map<String, String>... listOfMaps) {
```

Comment:
We call `listOfMaps(Collection)` here, which does the copying with `ListOfMapStringToStringCopier.copy`.